activerecord/lib/active_record/persistence.rb

```diff
@@ -222,10 +222,13 @@ def upsert(attributes, returning: nil, unique_by: nil)
       #   new records may considered duplicates according to different criteria. For MySQL,
       #   an upsert will take place if a new record violates _any_ unique constraint. For
       #   Postgres and SQLite, new rows will replace existing rows when the new row has the
-      #   same primary key as the existing row. By defining <tt>:unique_by</tt>, you can supply
-      #   a different key for matching new records to existing ones than the primary key.
+      #   same primary key as the existing row. In case of SQLite, an upsert operation causes
+      #   an insert to behave as an update or a no-op if the insert would violate
+      #   a uniqueness constraint. By defining <tt>:unique_by</tt>, you can supply
+      #   a different unique constraint for matching new records to existing ones than the
+      #   primary key.
       #
-      #   (For example, if you have a unique index on the ISBN column and use that as
+      #   (For example, if you have a unique index on the isbn column and use that as
```

Comment:
There are more occurrences of isbn that differ, can you reconcile them?

Reply:
To be precise, lets retain this, its being used same as `ID` vs `id`. Lets keep this usage

Reply:
I just wanted to change `ISBN column` to `isbn column` when it is explicitly referred as a column.

I think changing `ISBNs` to `isbns` will not look good therefore I will revert this change as well.

Reply:
Done.